src/Resources/public/Admin.js

```diff
@@ -206,6 +206,13 @@ var Admin = {
 
     add_filters: function(subject) {
         Admin.log('[core|add_filters] configure filters on', subject);
+
+        function updateCounter() {
+            var count = jQuery('a.sonata-toggle-filter .fa-check-square-o', subject).length;
```

Comment:
Depending on a CSS class for this looks fragile. Isn't there some underlying value you could leverage instead? Something more semantic?

Reply:
The full JS stuff is really messed up... It's hard to find a beautiful way

Reply:
Ok too bad, let's go with that then!

Reply:
Hope that @ju1ius will come back one day... He did great work to simplify that whole stuff